exec/java-exec/src/main/java/org/apache/drill/exec/physical/impl/validate/BatchValidator.java

```diff
@@ -173,6 +173,10 @@ public BatchValidator(ErrorReporter errorReporter) {
   }
 
   public static boolean validate(RecordBatch batch) {
+    // This is a handy place to trace batches as they flow up
+    // the DAG. Works best for single-threaded runs with few records.
```

Comment:
```suggestion
    // the DAG. Works best for single-threaded runs with a few records.
```

Reply:
Don't we have some print method for this case? So we don't need to make changes in code?

Reply:
So what the next steps?